src/main/java/uk/gov/hmcts/darts/arm/util/ArmResponseFilesUtil.java

```diff
@@ -9,13 +10,13 @@
 
 @UtilityClass
 @SuppressWarnings({"HideUtilityClassConstructor"})
+@Slf4j
```

Comment:
This is added but does not seem to be used?

Reply:
removed